MSAL/src/MSALPublicClientApplication.m

```diff
@@ -63,7 +63,7 @@ - (BOOL)generateRedirectUriWithClientId:(NSString *)clientId
         }
     }
     
-    MSAL_ERROR_PARAM(nil, MSALErrorRedirectSchemeNotRegistered, @"The required app scheme (%@) is not registered in the app's info.plist file.", scheme);
+    MSAL_ERROR_PARAM(nil, MSALErrorRedirectSchemeNotRegistered, @"The required app scheme (%@) is not registered in the app's info.plist file. Make sure the URI scheme matches exactly \"%@<clientID>\" format without any whitespaces.", scheme, scheme);
     
```

Comment:
Why is there <clientID> added to the end? Right now it will print "Make sure the URI scheme matches exactly "**msal3c62ac97-29eb-4aed-a3c8-add0298508da<clientID>**" format without any whitespaces.", which looks even more confusing to me.